saleor/account/i18n.py

```diff
@@ -38,6 +38,10 @@
     "zip": "ZIP code",
 }
 
+CountryAreaValues = namedtuple("CountryAreaValues", ["value", "choices"])
+
+COUNTRY_AREA_MAP = {"CH": CountryAreaValues("Graubünden", ["Graubünden"])}
```

Comment:
There should be another mapping nested per country, as multiple country areas will exist. The current structure supports only a single country area per country.